lib/Artifacts.php

```diff
@@ -74,6 +74,45 @@ public function getForPullRequest(int $id): array
 		return $this->mapPullRequestAssetsFromJson($validArtifacts, $id, $description);
 	}
 
+    public function getLatestMonthlyReport(): string
+    {
+        $query = <<<GRAPHQL
+        {
+          repository(owner: "LMMS", name: "lmms") {
+            discussions(first: 10, orderBy: {field: CREATED_AT, direction: DESC}) {
+              edges {
+                node {
+                  title
+                  bodyText
+                  category {
+                    name
+                  }
+                }
+              }
+            }
+          }
+        }
+        GRAPHQL;
+
+        try {
+            $results = $this->client->api('graphql')->execute($query);
+
+            // Process and filter discussions by tag
+            foreach ($results['data']['repository']['discussions']['edges'] as $result) {
+                $node = $result['node'];
+
+                // Check for "LMMS Progress Report:" string if progress report tag isn't available.
+                if ($node['category']['name'] === "progress report" || str_contains(strtolower($node["title"]), "lmms progress report:")) {
+                    return '## ' . $node['title'] . "\n" . $node['bodyText'];
```

Comment:
Rather than returning early here, I think we should show as many progress reports as possible in a dedicated "news" page or something of that nature.  We also should have a link for "more" that leads to the Discussions filtered on `progress report` so that people can go back further if they wish.